.github/workflows/fortran_packages.yml

```diff
@@ -28,5 +28,5 @@ jobs:
       - name: Upload data files
         uses: actions/upload-artifact@v3
         with:
-          name: _data
-          path: _data/
+          name: data
+          path: data/
```

Comment:
Can we keep this at `_data/`? The `data/` directory should contain the human generated files, which are updated with the repository and the `_data/` the machine generated once. The cache might be out of sync between `data/` and `_data/` directory, but changes to languages would become available immediately without refreshing the cache.

Reply:
Yes sir, sure will revert this changes.